server/base/src/main/java/org/apache/accumulo/server/master/balancer/HostRegexTableLoadBalancer.java

```diff
@@ -254,7 +255,7 @@ public int getMaxConcurrentMigrations() {
   }
 
   @Override
-  public void init(ServerConfiguration conf) {
+  public void init(ServerConfigurationFactory conf) {
```

Comment:
Sure, I've opened [ACCUMULO-4549](https://issues.apache.org/jira/browse/ACCUMULO-4549) to track the issue.  We can move discussion over there for how to move forward on a more permanent fix.